### DISCUSSION TITLE
github issues and projects experience

### DISCUSSION THREAD

**Discussion by warengonzaga** [community/community#6114] [Projects and Issues]
So far so good, combining table and board views is the more meaningful and the best way to manage multiple open source projects. 

I have a question regarding this new feature, is there a way to make it visible to everyone or is there a way to manage who can view my table list and board list?

**Comment by MylesBorins** [comment]
@WarenGonzaga I think you have put this on the wrong category, "Releases Feedback" is for the GitHub Releases feature. I think you likely want to us [issues feedback](https://github.com/github/feedback/discussions/categories/issues-feedback)

**Comment by michellemerrill** [comment]
@WarenGonzaga We moved this to the `issues` category.

**Reply by warengonzaga** [reply]
> @WarenGonzaga We moved this to the `issues` category.

Thanks for moving!
